Add Squishy65 keyboard

ramonimbao opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description

Add the Squishy65 keyboard.

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

drashna commented on keyboards/ramonimbao/squishy65/squishy65.c:
```diff
@@ -0,0 +1 @@
+#include "squishy65.h"
```
Could you add a license header here?

drashna commented on keyboards/ramonimbao/squishy65/squishy65.h:
```diff
@@ -0,0 +1,57 @@
+#pragma once
```
Could you add a license header, here?

ramonimbao commented:
Thanks! I've added it. I just copied it from my `wete` folder and renamed files, so it's missing in my `wete.c` and `wete.h`. Guess I should add a license to those too? Is it okay if I add it from this PR?

drashna commented:
No, that's fine.  The license header is something that we've recently started enforcing. 

And we have a link for the stuff we're trying to enforce, in general, actually.
https://docs.qmk.fm/#/pr_checklist


If you want to, you can open a separate PR for that keyboard.  (one keyboard per PR makes things easier for us to review, and for the CI to check)

fauxpark commented on keyboards/ramonimbao/squishy65/config.h:
```diff
@@ -0,0 +1,69 @@
+#define DEVICE_VER      0x0065
+#define MANUFACTURER Ramon Imbao
+#define PRODUCT Squishy65
+#define DESCRIPTION 65% keyboard
```
```suggestion
```

fauxpark commented on keyboards/ramonimbao/squishy65/keymaps/via/rules.mk:
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```
```suggestion
```

fauxpark commented on keyboards/ramonimbao/squishy65/rules.mk:
```diff
@@ -0,0 +1,20 @@
+MCU = STM32F072
+
+# Build Options
+#   comment out to disable the options.
+#
+
+#BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
+CUSTOM_MATRIX = no # Custom matrix file
+BACKLIGHT_ENABLE = no
+RGBLIGHT_ENABLE = yes
```
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```
